tests/accounts/test_accounts_profile.py

```diff
@@ -173,12 +183,22 @@ def test_admin_pop_up_console(accounts_base_url, selenium):
 @accounts
 def test_go_to_full_console(accounts_base_url, selenium):
```

Comment:
For this test, the `GIVEN` should be a logged in administrator; we're evaluating the full console link, not the log in process.